roles/tests-integration/.gitignore

```diff
@@ -0,0 +1 @@
+template-provider
```

Comment:
There is a single `.gitignore` in the project root used for all workspace crates